modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermEvaluator.java

```diff
@@ -248,8 +249,8 @@ private String _evaluateParentObjectDefinitionObjectFields(
 		throws PortalException {
 
 		ObjectDefinition objectDefinition = null;
-		ObjectField objectField2 = null;
-		String objectFieldName = null;
+		ObjectField relatedObjectField = null;
+		ObjectField relationshipObjectField = null;
```

Comment:
Since the name of the method has the `parent` word as prefix,
I would use it in the local variables:
```
ObjectDefinition parentObjectDefinition = null;
ObjectField parentObjectField = null;
ObjectField relationshipObjectField = null;
```

Reply:
Done